docs/pages/versions/unversioned/sdk/sqlite-next.mdx

```diff
@@ -0,0 +1,147 @@
+PRAGMA journal_mode = WAL;
+CREATE TABLE IF NOT EXISTS test (id INTEGER PRIMARY KEY NOT NULL, value TEXT NOT NULL, intValue INTEGER);
+INSERT INTO test (value, intValue) VALUES ('test1', 123);
+INSERT INTO test (value, intValue) VALUES ('test2', 456);
+INSERT INTO test (value, intValue) VALUES ('test3', 789);
+`);
+
+// `runAsync` is useful when you want to execute some write operations.
+const result = await db.runAsync('INSERT INTO test (value, intValue) VALUES (?, ?)', 'aaa', 100);
+console.log(result.lastInsertRowid, result.changes);
+await db.runAsync('UPDATE test SET intValue = ? WHERE value = ?', 999, 'aaa'); // Binding unnamed parameters from variadic arguments
+await db.runAsync('UPDATE test SET intValue = ? WHERE value = ?', [999, 'aaa']); // Binding unnamed parameters from array
+await db.runAsync('DELETE FROM test WHERE value = $value', { $value: 'aaa' }); // Binding named parameters from object
+
+// Prepare statement allows you compile your SQL query once and execute it multiple times with different parameters.
+const statement = await db.prepareAsync(
+  'INSERT INTO test (value, intValue) VALUES ($value, $intValue)'
+);
+await statement.runAsync({ $value: 'bbb', $intValue: 101 });
+await statement.runAsync({ $value: 'ccc', $intValue: 102 });
+await statement.runAsync({ $value: 'ddd', $intValue: 103 });
+await statement.finalizeAsync();
+
+// `getAsync` is useful when you want to get a single row from the database.
+const oneRow = await db.getAsync('SELECT * FROM test');
+console.log(oneRow.id, oneRow.value, oneRow.intValue);
+
+// `allAsync` is useful when you want to get all results as an array of objects.
+const allRows = await db.allAsync('SELECT * FROM test');
+for (const row of allRows) {
+  console.log(row.id, row.value, row.intValue);
+}
+
+// `eachAsync` is useful when you want to iterate SQLite query cursor.
+for await (const row of db.eachAsync('SELECT * FROM test')) {
+  console.log(row.id, row.value, row.intValue);
+}
+```
+
+### Executing queries within an async transaction
+
+```js
+const db = await SQLite.openDatabaseAsync('dbName');
+
+await db.transactionAsync(async () => {
+  const result = await db.getAsync('SELECT COUNT(*) FROM USERS');
+  console.log('Count:', result.rows[0]['COUNT(*)']);
+});
+```
+
+> **Note:** Please note that due to the nature of the async/await, We can't ensure that the transaction will be executed in the same order as the statements are written. A query out of a transaction may be executed inside the transaction. If you care about the order of execution, you should use the [`transactionExclusiveAsync`](#transactionexclusiveasynctask) function.
+
+### Executing PRAGMA queries
+
+```js
+const db = await SQLite.openDatabaseAsync('dbName');
+await db.execAsync('PRAGMA journal_mode = WAL');
+await db.execAsync('PRAGMA foreign_keys = ON');
+```
+
+> **Tip:** Enable WAL journal mode when you create a new database to improve performance in general. [Learn more](https://www.sqlite.org/wal.html).
```

Comment:
```suggestion
> **info** **Tip:** Enable [WAL journal mode](https://www.sqlite.org/wal.html) when you create a new database to improve performance in general.
```

- Let's use info type of callout to provide the tip here.
- Also, instead of adding learn more as context to provide link, we can just provide the link for the actual context.